logging/logging_test.go

```diff
@@ -19,6 +19,15 @@ func TestWithFields(t *testing.T) {
 	}
 }
 
+func TestWithWarnAsError(t *testing.T) {
```

Comment:
This currently does not involve `Warn` :)
Can we add a test that asserts that when set to `Error`, we still see `Warn` logs in the output?

Reply:
For sure. I just committed some of my latest changes to test for it. Check it out.